packing_product_change/models/sale.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp
```

Comment:
This module was created long time ago, please but the previous years before it

2010-2012, 2017